src/data/examples/zh-Hans/05_Image/04_Create_Image.js

```diff
@@ -1,20 +1,25 @@
 /*
- * @name Create Image
- * @description The createImage() function provides a fresh buffer of pixels to
- * play with. This example creates an image gradient.
+ * @name 创建图像
+ * @description createImage() 函数能让我们巧妙地操控一堆像素。 此范例创建了一个渐变图像。
  */
-let img; // Declare variable 'img'.
+let img; // 声明变量'img'
 
 function setup() {
   createCanvas(720, 400);
+  // 设置图像大小230*230 像素
```

Comment:
The `*` symbol should be replaced with `x` for consistency sake. Also it would be great if you can add a space before the first `230`.
```suggestion
  // 设置图像大小 230x230 像素
```